pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "titanoboa"
-version = "0.1.10b1"
```

Comment:
looks like the v0.1.10 tag had the wrong version: https://github.com/vyperlang/titanoboa/blob/v0.1.10/pyproject.toml#L3